### ISSUE TITLE
If you can create an awesome logo, please make a PR!

### ISSUE THREAD

**Issue by Kimmax** [cozyplanes/dB-UI#1]
Please make a PR or contact us at : <cozyplanes@tuta.io>

---

Hey @devBanner/frontend Team!
We need a logo for this project. I thought about something _like_ the devRant logo (No copy-pasta please), something that seems familiar, but is still our own.
You're the design gurus of course, if you think you have a better idea - no problem!
Looking forward to see your ideas :)

**Comment by afzalsayed96** [comment]
## Logo? How 'bout a marketing campaign 😎 


![devbanner-1_4](https://user-images.githubusercontent.com/14029371/33616549-348713ba-da03-11e7-87db-c5bbca776d1e.gif)

**Comment by Kimmax** [comment]
duuuuude
that's lit af
One proposal: Make it slower beginning and let it speed up. Might need to add some more banners tho
I can give you a copy of all generated ones up to now
Would only need a quick check if any of these is inappropriate 
Also maybe vertically center the text in the logo and justify "subtext"

**Comment by afzalsayed96** [comment]
I agree this is way too fast. Will add a few more banners as well. The subtext is left justified to match with the format of other devbanners.
Also the last frame of the GIF is what I propose for the logo.

**Comment by Kimmax** [comment]
I like the speed, actually. Just add some more banners and accelerate to that speed. Right now it's 100% throttle from the beginning. A exponential increase would look much smoother
I like the last frame as logo and while I get your point that it should match a banner, I think it'll look won't look right that way. It seems off..
If you won't mind I would like to see both versions against each other, if it's not too much work :)

**Comment by cozyplanes** [comment]
Make sure to add our collaborators' banner!

**Comment by afzalsayed96** [comment]
So I couldn't work today on the GIF but here's the logo horizontally and vertically aligned to center
[compact]
![devb logo-2](https://user-images.githubusercontent.com/14029371/33673631-543f91fc-dad3-11e7-9965-90ffecff9b76.png)
[spacious]
![devb logo-3](https://user-images.githubusercontent.com/14029371/33674313-21fc5b1a-dad5-11e7-884c-d5e6fc0e4a34.png)

We can also make one with transparent background in SVG and use it on background with various colors on the web

**Comment by Kimmax** [comment]
Yeees
I think the spacious one looks great!
For very small icons { dB } I think? I'm not that creative, if someone comes up with a better idea please drop it in here :D

**Comment by cozyplanes** [comment]
[db_icons.zip](https://github.com/devBanner/devBanner_Frontend/files/1567060/db_icons.zip)

Material design icons

@Kimmax

**Comment by LoganS1** [comment]
So we are going with @cozyplanes icons? If so I can create a PR which adds the icons to the repo so we can close this issue

**Comment by Kimmax** [comment]
Yes

**Comment by cozyplanes** [comment]
Even tho we are going with mine, not closing this cuz others can still talk.

**Comment by afzalsayed96** [comment]
[asset.zip](https://github.com/devBanner/devBanner_Frontend/files/1629429/asset.zip)


I noticed the image used in devbanner.center is low quality. The zip contains high quality png, svg and original AI file in case if needed for making any changes in future.

Cheers
skynet
